Pipfile.lock

```diff
@@ -38,7 +38,6 @@
                 "sha256:63579f9a0628e06278f7e47b7d7d5b6ce20dc65c5e96a6f3ca99a6adca0396e8"
             ],
             "index": "pypi",
-            "markers": "python_version >= '3.7' and python_version < '4.0'",
```

Comment:
Why are so many python_version checks gone? Was this something you did or something that `pipenv` did naturally?

Reply:
Pipenv did this, I couldn't explain why